### DISCUSSION TITLE
How to set the log retention policy?

### DISCUSSION THREAD

**Discussion by djpate** [errsole/errsole.js#108] [Q&A]
Hello, I would like to know how I can ask errsole to only keep n days worth of logs?

**Comment by SaiKumar2121** [comment]
hello @djpate,

You can easily set how long errsole retains logs. Just go to Settings in the dashboard, and under Data Retention, you'll find an option to set the "Logs TTL" (Time To Live). Simply enter the number of days you'd like to keep the logs (e.g., 30 days) and click Update. The logs older than that will automatically be deleted.

Check out the screenshot for reference:

![Screenshot from 2024-10-17 11-24-11](https://github.com/user-attachments/assets/c2fe221e-e69f-472c-a84e-acb7c90f6409)
